src/dscanner/highlighter.d

```diff
@@ -76,3 +80,48 @@ void writeSpan(string cssClass, string value)
 		stdout.write(`<span class="`, cssClass, `">`, value.replace("&",
 				"&amp;").replace("<", "&lt;"), `</span>`);
 }
+
+struct Theme
+{
+	string bg;
+	string fg;
+	string kwrd;
+	string com;
+	string num;
+	string str;
+	string op;
+	string type;
+	string cons;
+}
+
+immutable(Theme)* getTheme(string themeName)
+{
+	static immutable Theme solarized = {
```

Comment:
please don't use struct initializers like this, either use them to name your arguments (only needed when supporting old compilers) or use the auto generated struct ctor, which also can be used with named arguments (on newer compilers)

This syntax has been subject to deprecation before and I think once named arguments are widely supported it will quickly disappear